src/start.js

```diff
@@ -0,0 +1,61 @@
+import config from 'config';
+import cron from 'croner';
+
+import Archivist from './archivist/index.js';
+import logger from './logger/index.js';
+import Notifier from './notifier/index.js';
+import Tracker from './tracker/index.js';
+
+async function startOpenTermsArchive({ services, refilterOnly, schedule }) {
```

Comment:
In the context of the Open Terms Archive project, I am not in favour of adding the project name to that function name.

Since this is the exportable main loop, convention is to call this file `index.js`, and to export the default as the main function, which I suggest we call `track` 🙂